AngularVReact.md

```diff
@@ -1,5 +1,26 @@
 # Introduction
-Comparison points on Angular vs. React. Angular is full MVC while React is simply V.
+Before we get started, **I am in no way saying one framework is better than the other.** These are just some comparison points on Angular vs. React. In the end it all comes down to whatever you're looking for.
```

Comment:
This sentence feels unnecessary. What is we just remove all the winner comments?